java/bom/pom.xml

```diff
@@ -30,8 +30,45 @@ under the License.
   <artifactId>arrow-bom</artifactId>
   <version>17.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
+
   <name>Arrow Bill of Materials</name>
   <description>Arrow Bill of Materials</description>
+  <url>https://arrow.apache.org/</url>
+
+  <mailingLists>
+    <mailingList>
+      <name>Developer List</name>
+      <subscribe>dev-subscribe@arrow.apache.org</subscribe>
+      <unsubscribe>dev-unsubscribe@arrow.apache.org</unsubscribe>
+      <post>dev@arrow.apache.org</post>
+      <archive>https://mail-archives.apache.org/mod_mbox/arrow-dev/</archive>
```

Comment:
Could you use `lists.apache.org` directly such as https://lists.apache.org/list.html?dev@arrow.apache.org ?

Reply:
Done. I also added `github@arrow.apache.org` mailing list as well